src/main/java/Duke.java

```diff
@@ -47,34 +53,28 @@ public static void main(String[] args) {
                 taskArray[doneItem].isCompleted();
                 System.out.println(format);
                 System.out.println("Nice! I've marked this task as done:\n");
-                System.out.println("[" + taskArray[doneItem].getStatusIcon() + "] " + taskArray[doneItem].description + "\n");
+                System.out.println(taskArray[doneItem]);
                 System.out.println(format);
             }
 
             else if (stringArray[0].equals("todo")){ //ADDING A TODO
```

Comment:
Should the else if statement be in the same line as the closing curly bracket?

Reply:
Yup refer to https://se-education.org/guides/conventions/java/basic.html for coding standards